integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCrossDomainTransaction.java

```diff
@@ -550,30 +554,18 @@ private static void createDomain(String domainUid, String domainNamespace, Strin
       checkServiceExists(managedServerPrefix + i, domainNamespace);
     }
 
-    adminExtSvcRouteHost = createRouteForOKD(getExternalServicePodName(adminServerPodName), domainNamespace);
```

Comment:
why are we removing the tests changes made for OKD?

Reply:
now synced form main branch